src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1134,6 +1134,50 @@ internal bool HasIsByRefLikeAttribute(EntityHandle token)
             return null;
         }
 
+#nullable enable
+        internal UnmanagedCallersOnlyAttributeData? TryGetUnmanagedCallersOnlyAttribute(
```

Comment:
Consider adding a comment noting the importance of only loading the single attribute here.